RULES.md

```diff
@@ -0,0 +1,119 @@
+# Based Rules
+
+## Rationale
+
+These rules are implemented to ensure:
+- the repository stays neat and tidy,
+- a consistent Markdown style amongst all recipes,
+- consistent technical details amongst all recipes,
+- appropriate tags and images representative of the recipes on which they are included,
+- recipes are not condescending or verbose, and
+- only authentic, traditional, high-quality food.
+
+Make [based.cooking](https://based.cooking) based again!
```

Comment:
```suggestion
Keep [based.cooking](https://based.cooking) based!
```

Reply:
I understand the change in grammar because it implies that based.cooking was always based, however I intended it to be a parody of the orange man slogan.